ern-runner-gen-ios/src/hull/ErnRunner/AppDelegate.m

```diff
@@ -27,11 +27,12 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    // Start the container
+    // Configure the container
     ElectrodeContainerConfig *containerConfig = [[ElectrodeContainerConfig alloc] init];
     containerConfig.debugEnabled = RnDevSupportEnabled;
+    containerConfig.packagerHost = (PackagerHost)? PackagerHost : containerConfig.packagerHost;
+    containerConfig.packagerPort = (PackagerPort)? PackagerPort : containerConfig.packagerPort;
```

Comment:
I don't really understand these two lines logic.
Because we are inside the Runner and you import `RunnerConfig.h`, the `PackagerHost` and `PackagerPort` will always be defined, so I don't see the need of ternary here. 
And if they are not defined (won't be possible) then it does `containerConfig.packagerPort = containerConfig.packagerPort` which makes no real sense.
Could you clarify the need or just remove the ternary to just affect these variables with the values of `PackagerHost`/`PackagerPort` ?